### ISSUE TITLE
Homepage - Find Treatment should also set focus on facility search

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#297]
### Problem Description:
When using the keyboard for navigation, activating the Find Treatment CTA causes the page to scroll to the `#home-search` area with the `input` field in view. However, focus remains on the Find Treatment link. Focus should also shift to the area/element of intent.

### Solution:
In addition to the scrolling of the page, which is fine to do, focus should be set to either the area in question such as the `<div id="home-search" ... >` or the `input` for the location search itself (preferred). For the preferred solution, an ID will need to be added to the `input` to target.

![Screenshot of focus issue explanation](https://user-images.githubusercontent.com/1093423/64651509-f2219b80-d3de-11e9-8fd4-a579cf4fc48a.png)

**Comment by iamjolly** [comment]
I can't seem to add labels or assignees to this @Jkrzy, @allisonnorman, @hursey013. Do I need to be added to the SAMHSA team in GitHub, too?

This one needs the `accessibility` label. :)

**Comment by hursey013** [comment]
This has been addressed with https://github.com/18F/samhsa-prototype/pull/269/commits/b18237321563447eb5dfa619ffca9e73be8542ee
